server/services/licenceService.ts

```diff
@@ -151,6 +151,11 @@ export default class LicenceService {
     return this.licenceApiClient.createLicence(licence, user)
   }
 
+  async getlicenceStatistics(startDate: string, endDate: string, user: User) {
+    const data = await this.licenceApiClient.getlicenceStatistics(startDate, endDate, user)
```

Comment:
Are you converting from `d/mm/yyyy` -> `yyyy-mm-dd` somewhere?

Reply:
yes